### ISSUE TITLE
submission gave red flag even the solution was Accepted

### ISSUE THREAD

**Issue by frankbearzou** [raphaelheinz/LeetHub-3.0#29]
**Describe the bug**
the extension shows a red flag even the solution was Accepted.

**To Reproduce**
find a question that you already solved before. so the old code will retrieved by LeetCode. 
1. click reset button to clear the old code
2. write the code
3. click Submit button
4. wait... I didn't do anything... just wait
5. red flag

**Expected behavior**
Accepted solution should have green flag and submitted to GitHub. But now shows red flag without commit to Github.

**Screenshots**
![image](https://github.com/raphaelheinz/LeetHub-3.0/assets/11401804/abe5eca8-5192-46f5-9073-bd7f958a46e1)

**Comment by raphaelheinz** [comment]
Hey @frankbearzou,
I tried to reproduce this issue as you described, but pushing to GitHub worked on my side. Can you try to reproduce?

Unfortunately, the observers do not work 100%. Therefore, in rare cases the code may not be pushed to GitHub although submission to LeetCode was successful. I have not found a solution so far. I have described it in the README.

https://github.com/raphaelheinz/LeetHub-3.0/blob/ec9deb58e092b4a2b25075d49c17131b05f50bef/README.md?plain=1#L45-L51

We have added the manual synchronization button. You can use that on if such issues occur in the future. You can also try to reload the page.

**Comment by frankbearzou** [comment]
Is this because I always click Run button and get success result before I click Submit?

But the manual Push button did work me.

![image](https://github.com/raphaelheinz/LeetHub-3.0/assets/11401804/c0c08f93-e524-46dd-9d35-c9db7aa0d665)

**Comment by raphaelheinz** [comment]
Good to hear manual push button is working :)

It is fine to press "Run" first. It is only important to wait after you pressed "Submit". Do not click anything while the spinner is moving. I have again tried to reproduce but I had no luck. Can you describe more details? Does this issue always happen? 🤔 

Can you check the logs in Chrome console (opening using F12). If there is an issue with the script, the error may be printed there.

**Comment by frankbearzou** [comment]
I will try and add more logs. here is one.

![image](https://github.com/raphaelheinz/LeetHub-3.0/assets/11401804/c399e4de-7b84-4f9e-a9d6-7eee5b4be04c)

**Comment by raphaelheinz** [comment]
OK thanks, I will check

**Comment by raphaelheinz** [comment]
You are right @frankbearzou , the problem could be that it takes too long to submit the solution. The script polls for 10 seconds after pressing the "submit" button if the solution is valid. If it takes longer, pushing to GitHub is aborted.
How long does it take to submit after pressing the submit button on your side?

**Comment by frankbearzou** [comment]
if I press the the button, it will take less than 10 seconds. very strange.

**Comment by raphaelheinz** [comment]
Is this issue still persistent? Can we close?
